xdsl/dialects/scf.py

```diff
@@ -118,6 +119,48 @@ def get(
         return op
 
 
+@irdl_op_definition
+class ParallelOp(Operation):
+    name = "scf.parallel"
+    lowerBound: Annotated[VarOperand, IndexType]
+    upperBound: Annotated[VarOperand, IndexType]
+    step: Annotated[VarOperand, IndexType]
+    initVals: Annotated[VarOperand, AnyAttr()]
+    results: Annotated[VarOpResult, AnyAttr()]
+
+    body: Region
+
+    irdl_options = [AttrSizedOperandSegments()]
+
+    @staticmethod
+    def get(lowerBounds: Sequence[SSAValue | Operation],
+            upperBounds: Sequence[SSAValue | Operation],
+            steps: Sequence[SSAValue | Operation], body: Region):
```

Comment:
Can you add a `,` at the end of the arg list? Makes yapf format look much better.

I'd also welcome the type of body to be `Region | list[Block]`